kansha/card_addons/checklist/view.py

```diff
@@ -69,10 +74,9 @@ def render_Checklists(self, h, comp, model):
         with h.div(class_='checklists', id=id_):
             for index, clist in enumerate(self.checklists):
                 if can_edit:
-                    clist().init_input()
                     h << clist.on_answer(
                         lambda v, index=index: self.delete_checklist(index)
-                    ).render(h.AsyncRenderer())
+                    )
```

Comment:
If you remove the AsyncRender, all the checklists are re-rendered each time you add/remove/check an item in one of them and that causes flickering.